stories/Footer.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+
+import Footer from "../components/Footer/Footer"
+
+export default {
+	title: "Example/Footer",
+	component: Footer,
+} as ComponentMeta<typeof Footer>
+
+const Template: ComponentStory<typeof Footer> = (args) => <Footer />
```

Comment:
This component does not have props. We can remove the `args` from the function:

`const Template: ComponentStory<typeof Footer> = () => <Footer />`

Reply:
done, thanks!